travis/Dockerfile-travis-watch

```diff
@@ -1,4 +1,4 @@
-FROM mitodl/mm_watch_travis_yarn
+FROM mitodl/mm_watch_travis_yarn_update
```

Comment:
I'm going to undo that - I was thinking we might want to point things at a different container, but I don't think we need to.